docs/advanced-topics/tools-vs-workflows.rst

```diff
@@ -0,0 +1,117 @@
+Tools vs Workflows: Detailed Comparison
```

Comment:
This is the only colon in the sidebar. Suggest removing, which may (or may not) require some rewording